Tools/CISupport/ews-build/events.py

```diff
@@ -314,9 +315,41 @@ class GitHubEventHandlerNoEdits(GitHubEventHandler):
     ACTIONS_TO_TRIGGER_EWS = ('opened', 'synchronize')
     OPEN_STATES = ('open',)
 
+    @classmethod
+    def file_with_status_sign(cls, info):
+        if info.get('status') in ('added', 'modified'):
```

Comment:
do we know what are all the possible statuses?
If not, it's better to default to "+++" rather than "---" for unknown statuses (since +++ was cause the EWS to run based on FindModifiedLayoutTests).

Reply:
"moved" and "deleted" are the other obvious two. I couldn't find a documented place where they listed them all, and `---` is the safer of the two, because we have code that specifically invokes a layout test with `+++`. That behavior is only really valid when a test is added or modified, not in general.

Reply:
I would suggest to check for "moved", "deleted" and return `---`. Else return `+++` for anything else. This way, if we encounter any other status we would return `+++` and usually that would result in running more test-suites. In unexpected cases, it's better to run test-suites, rather than skipping them.

Reply:
The other way to think about this is that `+++` assumes that the file actually exists in the checkout, while `---` just assumes that file has been changed. If we're wrong and apply `+++` to a file that doesn't exist, our test invocation will (incorrectly) fail

Reply:
I think failing on unexpected status is better, so that we will become aware of that unexpected status and can handle it properly.

Reply:
Discussing offline with @jain-aakash, we're ok with treating a file like it exist if we get a weird status.